### PR TITLE
Revamped controls, beatified the board, added draw condition,

### PR DESCRIPTION
**Pull request by Corruption13**
Game loops till exit now as well.

All other basic functionality untouched.

When comparing code, in diff settings tab, near review changes button, click the ignore 'white space changes' tick box for more accurate comparison.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1,50 +1,73 @@
...
+'''
+while(True):
+    gameboard = {'7':' ','8':' ','9':' ',
+                 '4':' ','5':' ','6':' ',   # This is how the basic board looks like, analogue to the numpad.
+                 '1':' ','2':' ','3':' '}
+
+    print("\n"*100, "Welcome to Tic-Tac Toe! Use your numpad to input the blocks!")
+
+
+    def showboard():       #function prints the board - commend Sandeep for beautification.
```
Comment (pranavmodx) [inline main.py:13]: Please avoid unecessary commenting.

File: main.py [hunk 2]
```diff
@@ -1,50 +1,73 @@
...
+        print('|',gameboard['7'],'|',gameboard['8'],'|',gameboard['9'] , '|')
+        print('-------------')
+        print('|', gameboard['4'],'|',gameboard['5'],'|',gameboard['6'] ,'|')
+        print('-------------')
+        print('|', gameboard['1'],'|',gameboard['2'],'|',gameboard['3'], '|')
+        print('='*13)
+        print('\n')
+
+
+    def win_check(gb,t):    # Lazy brute force coded logic. - blame Pranav.
```
Comment (pranavmodx) [inline main.py:25]: It's just a temporarily implemented logic to work out the other parts of the program.

**Review by pranavmodx — COMMENTED** [review]
Suggestion : Try to avoid deleting heavy amount of code. I am afraid I can't see much changes among many blocks of code.

One example would be : 

Your Code
` def showboard():       #function prints the board - commend Sandeep for beautification.
         print('\n')
         print('='*13)
         print('|',gameboard['7'],'|',gameboard['8'],'|',gameboard['9'] , '|')
         print('-------------')
         print('|', gameboard['4'],'|',gameboard['5'],'|',gameboard['6'] ,'|')
         print('-------------')
         print('|', gameboard['1'],'|',gameboard['2'],'|',gameboard['3'], '|')
         print('='*13)
         print('\n')`

Earlier Code
`def showboard(board):
     print('\n')
     print(gameboard['topleft'],'|',gameboard['topmid'],'|',gameboard['topright'])
     print('---------')
     print(gameboard['midleft'],'|',gameboard['midmid'],'|',gameboard['midright'])
     print('---------')
     print(gameboard['bottomleft'],'|',gameboard['bottommid'],'|',gameboard['bottomright'])
     print('\n')`


Instead, modifying the code and making additions wherever necessary would be the order of the day - saves time as well as recognizes other's efforts.

File: main.py (hunk 1 above)
Comment (Corruption13) [reply main.py:13]: It's not unnecessary, we know the board looks like a numpad now. -.-

Comment (pranavmodx) [reply main.py:13]: `#function prints the board - commend Sandeep for beautification.`
_This_ doesn't seem particularly necessary and not the comment about numpad.

**Pull request merged**